db/migrate/20230821082856_create_registrations.rb

```diff
@@ -0,0 +1,10 @@
+class CreateRegistrations < ActiveRecord::Migration[7.0]
+  def change
+    create_table :registrations do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
AFAIK we don't need ```null: false``` here
Please check there is already a validation in rails to check null for associations unless ```optional: true``` is given